### ISSUE TITLE
A mistake about the network level parameters!

### ISSUE THREAD

**Issue by huizhang0110** [NoamRosenberg/AutoML#65]
In the origin paper, the network level update is 
<img width="325" alt="Screen Shot 2019-10-18 at 3 35 25 PM" src="https://user-images.githubusercontent.com/32508847/67078184-63652300-f1c3-11e9-9c7f-a0c80cb88091.png">

But in your code:
```python
           # normalized_betas[layer][ith node][0 : ➚, 1: ➙, 2 : ➘]
            for layer in range (len(self.betas)):
                if layer == 0:
                    normalized_betas[layer][0][1:] = F.softmax (self.betas[layer][0][1:].to(device=img_device), dim=-1)

                elif layer == 1:
                    normalized_betas[layer][0][1:] = F.softmax (self.betas[layer][0][1:].to(device=img_device), dim=-1)
                    normalized_betas[layer][1] = F.softmax (self.betas[layer][1].to(device=img_device), dim=-1)

                elif layer == 2:
                    normalized_betas[layer][0][1:] = F.softmax (self.betas[layer][0][1:].to(device=img_device), dim=-1)
                    normalized_betas[layer][1] = F.softmax (self.betas[layer][1].to(device=img_device), dim=-1)
                    normalized_betas[layer][2] = F.softmax (self.betas[layer][2].to(device=img_device), dim=-1)
                else :
                    `normalized_betas[layer][0][1:]` = F.softmax (self.betas[layer][0][1:].to(device=img_device), dim=-1)
                    normalized_betas[layer][1] = F.softmax (self.betas[layer][1].to(device=img_device), dim=-1)
                    normalized_betas[layer][2] = F.softmax (self.betas[layer][2].to(device=img_device), dim=-1)
                    normalized_betas[layer][3][:1] = F.softmax (self.betas[layer][3][:1].to(device=img_device), dim=-1)
```

**Comment by HankKung** [comment]
This section of code is the implementation of beta normalization before

> In the origin paper, the network level update is
> <img alt="Screen Shot 2019-10-18 at 3 35 25 PM" width="325" src="https://user-images.githubusercontent.com/32508847/67078184-63652300-f1c3-11e9-9c7f-a0c80cb88091.png">
> 
> But in your code:
> 
> ```python
>            # normalized_betas[layer][ith node][0 : ➚, 1: ➙, 2 : ➘]
>             for layer in range (len(self.betas)):
>                 if layer == 0:
>                     normalized_betas[layer][0][1:] = F.softmax (self.betas[layer][0][1:].to(device=img_device), dim=-1)
> 
>                 elif layer == 1:
>                     normalized_betas[layer][0][1:] = F.softmax (self.betas[layer][0][1:].to(device=img_device), dim=-1)
>                     normalized_betas[layer][1] = F.softmax (self.betas[layer][1].to(device=img_device), dim=-1)
> 
>                 elif layer == 2:
>                     normalized_betas[layer][0][1:] = F.softmax (self.betas[layer][0][1:].to(device=img_device), dim=-1)
>                     normalized_betas[layer][1] = F.softmax (self.betas[layer][1].to(device=img_device), dim=-1)
>                     normalized_betas[layer][2] = F.softmax (self.betas[layer][2].to(device=img_device), dim=-1)
>                 else :
>                     `normalized_betas[layer][0][1:]` = F.softmax (self.betas[layer][0][1:].to(device=img_device), dim=-1)
>                     normalized_betas[layer][1] = F.softmax (self.betas[layer][1].to(device=img_device), dim=-1)
>                     normalized_betas[layer][2] = F.softmax (self.betas[layer][2].to(device=img_device), dim=-1)
>                     normalized_betas[layer][3][:1] = F.softmax (self.betas[layer][3][:1].to(device=img_device), dim=-1)
> ```

This section of code is the implementation of beta normalization before further inference like you mentioned.
![image](https://user-images.githubusercontent.com/17802507/67079068-171ae280-f1c5-11e9-9c0a-f747fd53cd71.png)
